src/cli/Cli.ts

```diff
@@ -203,6 +203,23 @@ export class Cli {
       optionsWithoutShorts.options.output = this.getSettingWithDefaultValue<string | undefined>(settingsNames.output, undefined);
     }
 
+    // Validate the Option sets provided in the command
+    const optionsSets: string[][] | undefined = this.commandToExecute.command.optionSets();
+    const argsOptions: string[] = Object.keys(this.optionsFromArgs.options);
+    const cmdArgs = this.optionsFromArgs;
+    optionsSets?.forEach((optionSet) => {
+      if (optionSet.length > 0) {
+        const commonOptions = argsOptions.filter(opt => optionSet.includes(opt));
+        if (commonOptions.length === 0) {
+          return this.closeWithError(`Specify either ${optionSet.map(opt => opt).join(',')}.`, cmdArgs, false);
```

Comment:
Let's make this error message more human-readably by rephrasing to something like: _Specify one of the following options: a, b, c_. When using `either`, you'd also expect `or` and replacing the last `,` with an `or` adds unnecessary complexity.